.gitignore

```diff
@@ -21,3 +21,4 @@ auth/*.pem
 template
 build
 secrets.yml
+derek_*_key
```

Comment:
Should the gitignore match `derek-secret-key` pattern we moved to?

Reply:
For consistency, yes, though its not technically necessary as the local files can be named independently of the secret.